internal/streams/helpers.go

```diff
@@ -20,3 +20,11 @@ func ParseQuery(s string) url.Values {
 	}
 	return params
 }
+
+func RedactPassword(s string) string {
```

Comment:
Theoretically, this could be private since it's only used in this module so far.